### PR TITLE
update upgrade job bash to handle latest release sha

### PR DESCRIPTION
**Pull request by carolynhu**

### CODE REVIEW

File: upgrade/run_upgrade_test.sh [hunk 1]
```diff
@@ -48,8 +48,9 @@ function download_untar_istio_release() {
   # Download artifacts
   LINUX_DIST_URL="${url_path}/istio-${tag}-linux.tar.gz"
 
-  if [ "${tag}" == "master" ];then
-    GIT_SHA=$(curl "https://storage.googleapis.com/istio-build/dev/latest")
+  if [[ "${tag}" =~ "latest" ]];then
+    release_version=$(echo "${tag}" | cut -d'_' -f2)
```
Comment (jwendell) [inline upgrade/run_upgrade_test.sh]: Did you mean `-f1`?

Comment (carolynhu) [reply upgrade/run_upgrade_test.sh]: yes

**Comment by carolynhu** [comment]
> Does it make sense to also update defaults on lines 34 and 37?

Those values will be overwritten by prow jobs.

**Comment by carolynhu** [comment]
@jwendell could you please take another look? Thanks

**Pull request merged**